.github/workflows/lint-autoformat.yml

```diff
@@ -2,9 +2,11 @@ name: Apply lint suggestions
 
 on:
 
-  pull_request:
-    types: [opened, synchronize, reopened]
-    branches: [main]
+  push:
+    branches:
+      - main
```

Comment:
I don't think we need to push to main trigger, as lint should have been fixed there